docs/source/interledger.rst

```diff
@@ -0,0 +1,57 @@
+Interledger Lab
+===============
+
+.. note:: **Work in progress**. 
+
+
+Build the services:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml build
+
+Run RethinkDB in the background:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml up -d rdb
+
+
+Configure each ledger:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml run bdb-0 bigchaindb -y configure
+    $ docker-compose -f ledgers.yml run bdb-1 bigchaindb -y configure
+
+Initialize each ledger:
+
+.. code-block:: bash
+
+    $ docker-compose -f ledgers.yml run bdb-0 bigchaindb init
```

Comment:
this step can probably be skipped because `bigchaindb start` already does the init